form/create_custom_field_type.rst

```diff
@@ -67,14 +67,30 @@ check out the `ChoiceType`_ class.
     extend from ``AbstractType`` is only a convenience way of implementing the
     required ``FormTypeInterface``.
 
-There are three methods that are particularly important:
+There are a few methods that are particularly important:
 
 .. _form-type-methods-explanation:
 
+``getParent()``
```

Comment:
This description focuses on "what/how" instead of "when/why". It explains that this is *"like PHP inheritance but implemented in PHP, because you can't use real PHP inheritance"*.

But I'm missing information about the "why" and "when" to use this:

* Is it mandatory to define this method?
* Which value should return this method? (the FQCN of a class implementing FormType??)
* How should I use this? (e.g. if you create a form type similar to an existing one, extend from it ... otherwise, extend from the generic FormType ??)

Reply:
I've pushed a rewording.